### PR TITLE
Add instructions on running the MC/DC checker tool

### PR DESCRIPTION
**Pull request by iche033**
Signed-off-by: Ian Chen <ichen@osrfoundation.org>

Added brief overview of what the MC/DC checker does and instructions on how to manually run it in your own workspace. 

I also added some instructions on generating coverage data for a ROS package.

### CODE REVIEW

File: source/How-To-Guides/MC-DC-Testing-Tool.rst [hunk 1]
```diff
@@ -1,7 +1,140 @@
...
+
+* Some non-tree BDDs do not have solutions
+
+  Looking at the console output produced by the ``mcdc_checker`` tool, the ``bdd_is_not_tree_like occurred in:`` section lists issues with non-tree BDDs. However, it is possible that not all of them contain solutions.
+
+
+How to get coverage data for a specific ROS package
+---------------------------------------------------
+
+Here are instructions for generating coverage data with the ``gcov`` and ``lcov`` coverage tools, using the ``rcpputils`` pacakge as an example.
```
Comment (ivanperez-keera) [inline source/How-To-Guides/MC-DC-Testing-Tool.rst]: typo: package

File: source/How-To-Guides/MC-DC-Testing-Tool.rst [hunk 2]
```diff
@@ -1,7 +1,140 @@
...
+
+  The checker invokes ``clang`` to preprocess the input files. If include files are not found, it will not be able to do ``#include`` file expansion. Underneath the hood, the checker runs:
+
+  .. code-block:: bash
+
+     clang input_file.c -E > output
+
+  where ``-E`` means run preprocesor only.
+
+  You can pass include directories as arguments to the mcdc_checker tool, e.g
```
Comment (ivanperez-keera) [inline source/How-To-Guides/MC-DC-Testing-Tool.rst]: Formatting: \`\`mcdc_checker\`\`

File: source/How-To-Guides/MC-DC-Testing-Tool.rst (hunk 1 above)
Comment (iche033) [reply source/How-To-Guides/MC-DC-Testing-Tool.rst]: fixed. [7eb530d](https://github.com/space-ros/docs/pull/13/commits/7eb530da49fdaab0c8813c90a2340c836e8292b6)

File: source/How-To-Guides/MC-DC-Testing-Tool.rst (hunk 2 above)
Comment (iche033) [reply source/How-To-Guides/MC-DC-Testing-Tool.rst]: updated. [7eb530d](https://github.com/space-ros/docs/pull/13/commits/7eb530da49fdaab0c8813c90a2340c836e8292b6)

**Review by nuclearsandwich — APPROVED** [review]

**Review by mjeronimo — APPROVED** [review]

**Pull request merged**